cfme/tests/ansible/test_embedded_ansible_basic.py

```diff
@@ -150,6 +150,29 @@ def test_embedded_ansible_repository_crud(ansible_repository, wait_for_ansible):
     assert view.description.value == updated_description
 
 
+@pytest.mark.rhel_testing
+@pytest.mark.tier(1)
+def test_embedded_ansible_repository_invalid_url_crud(request, appliance, wait_for_ansible):
+    """
+    Polarion:
+        assignee: sbulage
+        casecomponent: Ansible
+        caseimportance: critical
+        initialEstimate: 1/6h
+        tags: ansible_embed
+    """
+    repositories = appliance.collections.ansible_repositories
+    repository = repositories.create(
+        name=fauxfactory.gen_alpha(),
+        url='https://github.com/sbulage/invalid_repo_url.git',
+        description=fauxfactory.gen_alpha())
+    view = navigate_to(repository, "Details")
+    assert view.entities.summary("Properties").get_text_of("Status") == "failed"
+
+    if repository.exists:
```

Comment:
Optional:
As Repository class using `BaseEntity` you can use `delete_if_exists`.

Reply:
@digitronik Added `delete_if_exists`